### DISCUSSION TITLE
Terminal

### DISCUSSION THREAD

**Discussion by DrCrocs247** [community/community#23300] [New to GitHub]
<p>Please can I open a terminal on my phone. A web terminal I can use to WORK with github repository</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/drcrocs247">@drcrocs247</a>,</p>
<p>Thanks for being here! You can use the GitHub mobile application. For more information and to download see: <a href="https://github.com/mobile/" rel="nofollow noopener">https://github.com/mobile/</a></p>
